interface/src/app/analyze/analysis/analysis.js

```diff
@@ -38,12 +48,7 @@ function AnalysisCtrl($scope, $log, $q, $state, SampleBin) {
       {'typename': 'medium'},
       {'typename': 'temperature'},
       {'typename': 'treatment'}
-    ],
-    updateMlModel: function(value) {
-      if (value !== null && value.id !== null) {
-        SampleBin.getActivityForSampleList(value.id);
-      }
-    }
+    ]
```

Comment:
Ugh. I'm getting `npm` errors trying to get my local clone of your dev branch to build.

@dongbohu: Can you try something on your instance to see if it helps? Let me point out the line I'd like to change...